test/export/test_export.py

```diff
@@ -6177,7 +6177,7 @@ class M(torch.nn.Module):
             def forward(self, x, y):
                 a = x.item()
                 torch._check_is_size(a)
-                torch._check(a <= y.size(0))
+                torch._check(a < y.size(0))
```

Comment:
I had to do this because with the former check, we couldn't tell whether `a >= y.size(0)`, creating a new guard.

Reply:
Reverted. Not needed after replacing the guards tweak with the static checks.